atomic_reactor/plugins/exit_store_metadata_in_osv3.py

```diff
@@ -207,7 +207,7 @@ def run(self):
             self.log.debug("annotations: %r", annotations)
             raise
 
-        labels = self.make_labels()
+        labels = self.make_labels(metadata)
```

Comment:
It calls update_labels_on_build(), which takes a delta not the full set.

Thanks, I think that's a better approach.